ProcessMaker/Http/Controllers/ProcessController.php

```diff
@@ -246,11 +246,10 @@ public function destroy(Process $process) // destory existing process to DB / UI
 
     public function triggerStartEventApi(Process $process, Request $request)
     {
-        $api_request = new ApiProcessController();
-        $response = $api_request->triggerStartEvent($process, $request);
-        $instance_id = $response->data['_request']['id'];
+        $apiRequest = new ApiProcessController();
+        $response = $apiRequest->triggerStartEvent($process, $request);
 
-        return redirect('/requests/' . $instance_id);
+        return redirect('/requests/' . $response->id);
```

Comment:
I agree with this change.

About the magic variable `data['_request']` since Nayra-engine runs always in async mode (including the Start Event), this magic variable will not be populated immediatly. Right now, the in-core Engine runs the Start Event using a `distpachNow` but this was deprecated in Laravel 8.

Considering that scenario in this case is right to use `$request->id` since this is the only one value required from `_request`.